libbeat/docs/version.asciidoc

```diff
@@ -5,4 +5,4 @@
 :python: 2.7.9
 :docker: 1.12
 :docker-compose: 1.11
-:branch: 6.x
+:branch: 6.2
```

Comment:
doc-branch also needs to be 6.2. That's the attribute used in 99% of the Beats docs.

The `branch` attribute was added recently when you integrated the X-Pack content here https://github.com/elastic/beats/commit/08f9be04058782c31a5774f5b74b91f4aa0fbc95. You reverted some of the changes. Is {branch} even used anywhere?

Reply:
Oh I see. It's the disconnect between the shared attributes and the ones I use for Beats. I'll add a cleanup issue for this. Just make sure you bump the doc-branch version.

Incidentally, I think doc-branch is more descriptive